tests/omhttp-basic.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# This file is part of the rsyslog project, released under ASL 2.0
+
+#  Starting actual testbench
+. ${srcdir:=.}/diag.sh init
+
+port="$(get_free_port)"
+omhttp_start_server $port
+
+error_file=$(pwd)/$RSYSLOG_DYNNAME.omhttp.error.log
+rm -f $error_file
+
+generate_conf
+add_conf '
+#$DebugLevel 2
```

Comment:
we don't need the debug level here. You can turn on debugging via `diag.sh`, see https://github.com/rsyslog/rsyslog/blob/master/tests/diag.sh#L75